### PR TITLE
Fix document step by step guide

### PR DESCRIPTION
**Pull request by liangruibupt**
*Issue #2 Issue #4  Issue #5 Issue #6 #7 #8 Issue #12 *

*Description of changes:*
Update the document for more and clear step by step guide

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.DS_Store
+china/.DS_Store
```
Comment (stevensu1977) [inline .gitignore:2]: please remove .DS_Store

File: china/2020_EKS_Launch_Workshop/步骤4-部署微服务以及配置ALBIngressController.md [hunk 2]
```diff
@@ -106,9 +106,9 @@ eksctl utils associate-iam-oidc-provider --cluster=${CLUSTER_NAME} --approve --r
  * 请注意官方的policy里面包含了WAF等服务，中国区没有所以需要手动删除,修改好的已经放在resource/alb-ingress-controller目录下
 
 ```bash
-cd resource/alb-ingress-controller
+cd china/2020_EKS_Launch_Workshop/resource/alb-ingress-controller
```
Comment (stevensu1977) [inline china/2020_EKS_Launch_Workshop/步骤4-部署微服务以及配置ALBIngressController.md]: I prefer all step guide lab context is china/china/2020_EKS_Launch_Workshop,
so, we only use kubectl apply -f resource/<lab step>/xxx.yaml / or <lab step>/xxxx.yaml.

File: .gitignore (hunk 1 above)
Comment (stevensu1977) [reply .gitignore:2]: you can ignore .gitignore  self

File: china/2020_EKS_Launch_Workshop/步骤1-准备实验环境.md [hunk 3]
```diff
@@ -68,9 +68,9 @@ sudo mv ./eksctl /usr/local/bin
 #kubectl
 https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html
 #macOS
-curl -o kubectl https://amazon-eks.s3-us-west-2.amazonaws.com/1.15.10/2020-02-22/bin/darwin/amd64/kubectl
```
Comment (stevensu1977) [inline china/2020_EKS_Launch_Workshop/步骤1-准备实验环境.md]: this is fix md format issue ? Sorry, I'm don't get your point   :<

**Review by stevensu1977 — COMMENTED** [review]
if you can rebase your pull request form new origin master ,  that's LGTM.

File: china/2020_EKS_Launch_Workshop/步骤1-准备实验环境.md (hunk 3 above)
Comment (liangruibupt) [reply china/2020_EKS_Launch_Workshop/步骤1-准备实验环境.md]: It is typo.  I have fixed it.

**Comment by stevensu1977** [comment]
Thank your PR !

**Pull request merged**